igvc_gazebo/config/igvc_joint_control.yaml

```diff
@@ -6,9 +6,19 @@
   # effort Controllers ---------------------------------------
   left_wheel_effort_controller:
     type: effort_controllers/JointEffortController
-    joint: axle_to_left_wheel
+    joint: left_axle
```

Comment:
Are these PID values for a different process than the ones set up in motor_controller.launch? if not, they should be the same

Reply:
These actually do not do anything